project_event/report/project_task_activity_report.xml

```diff
@@ -80,7 +80,12 @@
                 <td> <t t-esc="task['department']"/></td>
                 <td> <t t-esc="task['employee']"/></td>
                 <td> <t t-esc="datetime.datetime.strptime(task['expected_start'], '%Y-%m-%d %H:%M:%S').strftime('%H:%M')"/></td>
-                <td> <t t-esc="datetime.datetime.strptime(task['real_start'], '%Y-%m-%d %H:%M:%S').strftime('%H:%M')"/></td>
+                <t t-if="task['real_start']"> 
```

Comment:
What about the other fields? Do they need check?

Reply:
No because they are required fields

Reply:
I will add tests for this possible arising issue